plugin-api/src/main/java/org/hyperledger/besu/plugin/services/metrics/PoAMetricsService.java

```diff
@@ -21,7 +21,11 @@
 
 /**
  * Provides relevant data for producing metrics on the status of a Proof of Authority (PoA) node.
+ *
+ * @deprecated This interface has been replaced by {@link
+ *     org.hyperledger.besu.plugin.services.query.PoaQueryService}
  */
+@Deprecated
 public interface PoAMetricsService {
```

Comment:
nit: Can we check if we're using `PoA` or `Poa` to make sure these new APIs match.  I think `Poa` is strictly right according to the style guide but looks really weird to me - I'd expect `PoA`. I can live with it either way, just want to minimise inconsistency.

Reply:
I _personally_ am in favour of Poa, then POA then finally PoA (but it doesn't conform to any recognised capitalisation that I can think of)